src/featureLayer.js

```diff
@@ -200,15 +214,18 @@ var featureLayer = function (arg) {
   };
 
   /**
-   * Free all resources
+   * Free all resources.
    */
   this._exit = function () {
     m_this.clear();
     s_exit();
   };
 
   /**
-   * Draw
+   * Draw.  If the layer is visible, call the parent class's draw function and
+   * the renderer's _render function.
```

Comment:
I am thinking that mentioning a private method might be less useful. We could just state the this calls the renderer render function.